src/scheduler/README.md

```diff
@@ -0,0 +1,40 @@
+# Scheduler framework
+Schedules flows for execution.
+
+## Default implementation is to be implemented yet
+There are 3 objects that a `Scheduler` instance requires: `config`, `flowsDao` and `schedulePublisher`. These objects have to be provided to the constructor of `Scheduler`.
+`flowsDao` is a service, where `Scheduler` finds flows for execution.
+`schedulePublisher` is a service which is supposed to publish a command for scheduling a certain flow.
+
+### Elastic.io implementation
+As an example, `elastic.io` implementation of the mentioned dependencies:
+
+```
+const flowScheduler = new FlowScheduler({ amqpConnection: init.getAMQPConnection() });
```

Comment:
I don't sure that that is good example. It is too deeply integrated with elastic.io specific.
I think the best possible example is code => use old code that works with k8s CRD's for dao and src/backendCommonsLib/src/QueueCreator.js as schedulePublisher